R/install-git.r

```diff
@@ -69,3 +76,44 @@ remote_metadata.git_remote <- function(x, bundle = NULL, source = NULL) {
     RemoteSha = sha
   )
 }
+
+remote_sha.git_remote <- function(remote, ...) {
+  res <- system(
```

Comment:
Is it possible to do this with git2r? devtools traditionally hasn't assumed that you have git installed.